apps/documentation/src/app/app.ts

```diff
@@ -26,7 +26,7 @@ import { SideNavigationComponent } from './components/side-navigation/side-navig
     </div>
   `,
 })
-export class AppComponent {
+export class App {
```

Comment:
question: Is there a migration script for this?

Reply:
Not an official one, I did write one myself to do most of these changes. I think I deleted it since I had finished with it, but if you want it I may be able to restore it.

Reply:
Yes, please! Maybe a gist would be simpler for you?

Reply:
No problem, I'll see if I can recover it tomorrow and I'll share it with you 😊

Reply:
Here's a gist containing the generator code:
https://gist.github.com/ashley-hunter/9c38fcb133d8a776eff461fa1bdd0645

Your mileage may vary on how good it is 😊 but it may help a bit.

Reply:
Thank you! You're the best!

Reply:
😂 thank you! Hope it helps!